plone-5.0.x.cfg

```diff
@@ -1,4 +1,8 @@
 [buildout]
+extends =
+    https://raw.github.com/collective/buildout.plonetest/master/test-5.0.x.cfg
+    buildout.cfg
+
 test-eggs +=
     plone.app.contenttypes
```

Comment:
yes it is needless, but I did leave it there as it was there before and has noting to do with this pull request.